layouts/_default/baseof.css

```diff
@@ -11,6 +11,7 @@
   --content-max-width: 1280px;
   --content-padding: 1rem;
   --border: 1px solid #ccc;
+  --outline: 2px dotted rgba(0, 0, 0, 0.5);
```

Comment:
I would have named this `--focus-outline` but we don`t use outline for anything else (I think), so this is great. Change the name if you think my suggestion is good, but it's up to you.

```suggestion
  --focus-outline: 2px dotted rgba(0, 0, 0, 0.5);
```

Reply:
--focus-outline sounds better so I changed this.